RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -544,6 +544,106 @@ class PumpOpsSynchronous {
         }
         return frameData as Data
     }
+    
+    internal func getGlucoseHistoryEvents(since startDate: Date) throws -> ([TimestampedGlucoseEvent], PumpModel) {
+        try wakeup()
+        
+        let pumpModel = try getPumpModel()
+        
+        var events = [TimestampedGlucoseEvent]()
+        
+        // Going to scan backwards in time through events, so event time should be monotonically decreasing.
+        let eventTimestampDeltaAllowance = TimeInterval(hours: 9)
```

Comment:
There are out of order events due to relative timestamping. I changed the timestamp check on https://github.com/ps2/rileylink_ios/pull/270/commits/f266ebbf41e9888470d9db92285018331b2afda4#diff-bf2d04fbde756ea81dd46dafafbdacddR589 to use the reference timestamped events only. That should ensure that all relevant glucose events are captured since startDate at the expense of maybe capturing a few extra independently timestamped (non-sgv) entries.